packages/neuron-shared/tests/load-env.test.ts

```diff
@@ -0,0 +1,46 @@
+import { loadEnv } from "../src/load-env"
+import fs from 'fs'
+
+
+jest.mock('fs')
+describe("Load Env", () => {
+  beforeAll(() => {
+    fs.readFileSync = jest.fn().mockImplementation((filepath) => {
+      if (filepath === '.env.test.local') {
+        return 'TEST_ENV="from .env.test.local"'
+      } else if (filepath === '.env.test') {
+        return 'TEST_ENV="from .env.test"'
+      } else if (filepath === '.env') {
+        return 'TEST_ENV="from .env"'
+      } else {
+        return ''
+      }
+    })
+  })
+  afterAll(() => {
+    jest.resetAllMocks()
+  })
+  describe("read one by one", () => {
```

Comment:
`it("no matched .env files", () => {})` should be covered to ensure no unexpected files will be loaded when there aren't any `.env` files